docs/features/plugins/stable_diffusion.md

```diff
@@ -45,9 +45,17 @@ To use Stable Diffusion with this project, you will need to download and install
 ## 3. Run Stable Diffusion (either .sh or .bat file according to your operating system)
 
 ## 4. In the app, select the plugins endpoint, open the plugins store, and install Stable Diffusion
-    - You will need the stable diffusion webui API URL, which should be `http://127.0.0.1:7860`
-    - Alternatively: you (the admin) can set the value in `\.env` to bypass the prompt
-        - `SD_WEBUI_URL=http://127.0.0.1:7860`
+### Local Install
+- You will need the stable diffusion webui API URL, which should be `http://127.0.0.1:7860`
+- Alternatively: you (the admin) can set the value in `\.env` to bypass the prompt
+  - `SD_WEBUI_URL=http://127.0.0.1:7860`
+### Docker Install
+- Use `SD_WEBUI_URL=http://host.docker.internal:7860` in the `.env` file or in the plugins store
+- Uncomment the following in `docker-compose.yml` file:
+```yaml
+ extra_hosts: # if you are running APIs on docker you need access to, you will need to uncomment this line and next
```

Comment:
can you test if it still works on windows with this setting on? it may only be an issue for linux users that requires it uncommented.

Reply:
good call, I'll check that right now

Reply:
leave it pending for now
unrelated bugs are preventing me to complete the test at the moment

Reply:
What are the unrelated bugs?

Reply:
(cleared cache and cookie, tried incognito mode, tested on 2 fresh install to make sure it wasn't a glitch on my end)

local install
npm ci error: 'test' is not recognized as an internal or external command

docker install:
no user, user registration was closed even if marked as true in the .env

UI/Endpoint:
invalid api key for plugin endpoint when setting OPENAI_API_KEY from UI

plugin store:
cannot get the SD plugins to show option to enter api url fom plugin store
install/uninstall bug when cleared chache and cookie (see screenshot)

![image](https://github.com/danny-avila/LibreChat/assets/32828263/6125576b-a652-4c5a-a9da-37285982acf6)

![image](https://github.com/danny-avila/LibreChat/assets/32828263/4b333a6c-9293-4828-8de3-bbd96a15f401)

Reply:
I was able to test on an install that was not up to date:

I had an error with cuda in SD but watching the sd console output I was able to see that the request was properly sent, it seems to be working.

(I had an unrelated cuda error , it seems that I broke my sd install after setting up cuda for something else yesterday)

Reply:
I will retest with latest updates and resolve if working

Reply:
![image](https://github.com/danny-avila/LibreChat/assets/32828263/c786ce15-1c3a-4cd4-8e5f-37806b9278c6)

Reply:
on windows, uncommenting the two lines in docker-compose.yml, using "http://host.docker.internal:7860" as provided value from the store UI, everything worked properly